hydra/functions/FlatteLineShape.h

```diff
@@ -238,7 +240,7 @@ for(size_t i = 0; i < fParams.size() ; i++) {
         double fBachelorMass;
         double fMotherMass;
         double fRadi;
-        std::vector<std::vector<double>> fParams;
+        std::array<std::array<double,3>,2> fParams;
```

Comment:
This member, `std::array<std::array<double,3>,2> fParams;` is not copyable in CUDA. AFAIK, there is no implementation of std::array in CUDA.  Please, use `double fParams[6]`